### ISSUE TITLE
Add ability to create product image with meatafield

### ISSUE THREAD

**Issue by vleontyev** [nozzlegear/ShopifySharp#180]
In Shopify API, you can create image together with metafields in one API call. You cannot do it with ShopifySharp

```
POST /admin/products/#{id}/images.json
{
  "image": {
    "position": 1,
    "metafields": [
      {
        "key": "new",
        "value": "newvalue",
        "value_type": "string",
        "namespace": "global"
      }
    ],
    "attachment":  ....
```

**Comment by robgregory** [comment]
I'm using v 4.10.2 but getting the following exception: 'ProductImage' does not contain a definition for 'Metafields'

**Comment by dkershner6** [comment]
This is covered in https://github.com/nozzlegear/ShopifySharp/pull/265

**Comment by nozzlegear** [comment]
Published in 4.15.2, thanks to @dkershner6
